test/cypress/e2e/search.spec.js

```diff
@@ -18,29 +17,13 @@ describe('Search', () => {
 
   function assertAccountPage(accountsAddress) {
     cy.wait('@requestAccount');
-    cy.get(ss.accountAddress).should('have.text', accountsAddress)
-      .and(() => {
-        expect(getSearchesObjFromLS()[0].id).to.equal(accountsAddress);
-        expect(getSearchesObjFromLS()[0].searchTerm).to.equal(accountsAddress);
-      });
+    cy.wait('@requestDelegate');
```

Comment:
The function name describes the purpose of checking if we are at the account page. For that, we are searching for an element specific to that page. Putting here the search bar elements you are not testing anything

Reply:
fixed